pygeppetto/model/datasources/datasources.py

```diff
@@ -144,6 +144,7 @@ class DataSource(Node):
     dependenciesLibrary = EReference(ordered=True, unique=True, containment=False, upper=-1)
     targetLibrary = EReference(ordered=True, unique=True, containment=False)
     fetchVariableQuery = EReference(ordered=True, unique=True, containment=True)
+    auth = EAttribute(eType=EString, derived=False, changeable=True)
```

Comment:
@rodriguez-facundo this is not a change we can make here as this code is automatically generated from the ecore specification.  Usually those changes are made by @tarelli in the Java side.